### PR TITLE
govcms 1.3.0 satis update

### PR DESCRIPTION
**Pull request by simesy**
Mostly Toby's work

### CODE REVIEW

File: satis-config/govcms-develop.json [hunk 1]
```diff
@@ -9,17 +9,11 @@
         { "type": "vcs", "url": "https://github.com/govcms/require-dev" }
     ],
     "require": {
-        "govcms/govcms": "1.x-dev",
+        "govcms/govcms": "dev-develop",
```
Comment (tobybellwood) [inline satis-config/govcms-develop.json]: this doesn't resolve - I preferred to use a branch reference here (e.g. dev-release/1.2.0) to reinforce that the develop branch can be pointed to a future state of master - given that it is the release strategy for GovCMS to merge to a branch for testing - so the next release/* branch becomes a de-facto "develop"

File: scripts/check-missing.sh [hunk 2]
```diff
@@ -30,17 +34,23 @@ rm -Rf vendor && rm -Rf web/core && rm -Rf web/modules/contrib/* && rm -Rf web/p
 cp composer.json composer-copy.json && cat composer-copy.json \
   | jq .repositories='{"drupal":{"type":"composer","url": "https://packages.drupal.org/8"},"custom":{"type":"path","url":"custom/composer"},"govcms":{"type":"composer","url":"http://localhost:4142/'"${BRANCH}"'"}}' \
   | tee composer.json > /dev/null
-# Point composer.json the appropriate branch versions if testing develop or master.
+
+# @todd: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
```
Comment (tobybellwood) [inline scripts/check-missing.sh]:
```suggestion
# @todo: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
```

File: scripts/verify.sh [hunk 3]
```diff
@@ -31,10 +33,15 @@ composer config repositories.govcms composer http://localhost:4141/"${BRANCH}"
 echo -e "\033[1;35m--> Repositories updated...\033[0m"
 composer config repositories | jq .
 
+# @todd: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
```
Comment (tobybellwood) [inline scripts/verify.sh]:
```suggestion
# @todo: remove once govcms/govcms no longer requires "symfony/event-dispatcher:v4.3.11 as v3.4.35" which only works at the root composer.json level.
```

File: scripts/verify.sh [hunk 4]
```diff
@@ -14,10 +14,12 @@ else
 fi
 GOVCMS_VERSION=$(cat ./satis-config/${config} | jq -r '.require."govcms/govcms"')
```
Comment (tobybellwood) [inline scripts/verify.sh:15]: Missing GOVCMS_VERSION_SCAFFOLD_TOOLING and GOVCMS_VERSION_REQUIRE_DEV in here

```suggestion
GOVCMS_VERSION=$(cat ./satis-config/${config} | jq -r '.require | .["govcms/govcms"]')
# Versions needed for stable testing.
GOVCMS_VERSION_SCAFFOLD_TOOLING=$(cat ./satis-config/"${config}" | jq -r '.require | .["govcms/scaffold-tooling"]')
GOVCMS_VERSION_REQUIRE_DEV=$(cat ./satis-config/"${config}" | jq -r '.require | .["govcms/require-dev"]')
```

**Comment by tobybellwood** [comment]
This looks good, couple of minor todo nits, variable add and a question about the develop branch, but the logic in the scripts is better

File: scripts/verify.sh (hunk 4 above)
Comment (simesy) [reply scripts/verify.sh:15]: Ahh i thought my verify passed, it clearly didn't. Un momento.

**Comment by simesy** [comment]
I've fixed the verify.sh script, including the variables.
After a couple of mis-fires, i have this and I believe it works as god intended.
For develop:
```
        "govcms/govcms": "1.x-dev",
        "govcms/scaffold-tooling": "dev-develop",
        "govcms/require-dev": "dev-develop",
```
and for master:
```
        "govcms/govcms": "1.x-dev",
        "govcms/scaffold-tooling": "dev-master",
        "govcms/require-dev": "dev-master",
```

**Comment by simesy** [comment]
Sorry the changes weren't pushed, now they are

**Review by tobybellwood — APPROVED** [review]
thanks for the tinkering, looks good

**Comment by simesy** [comment]
Maybe a squash merge and credit @tobybellwood, I really just iterated off his legwork.

**Pull request merged**